src/inspector_agent.cc

```diff
@@ -31,7 +31,7 @@ namespace {
 const char DEVTOOLS_PATH[] = "/node";
 
 void PrintDebuggerReadyMessage(int port) {
-  fprintf(stderr, "Debugger listening on port %d. "
+  fprintf(stderr, "Debugger listening on port %d. \n"
```

Comment:
If there’s a line break here anyway, the preceding space might as well be removed, no?